### PR TITLE
Moves and renames `PhotoFunctions::exists` to `Photo::isDuplicate`.

### PR DESCRIPTION
**Pull request by hermzz**

### CODE REVIEW

File: app/Photo.php [hunk 1]
```diff
@@ -35,6 +35,21 @@ public function owner()
...
 
 
+    /**
+     * @param string $checksum
+     * @param $photoID
+     * @return array|false Returns a subset of a photo when same photo exists or returns false on failure.
+     */
+    public function isDuplicate(string $checksum, $photoID = null)
+    {
+        $sql = $this->where('checksum', '=', $checksum);
```
Comment (ildyria) [inline app/Photo.php]: I am not sure about the `$this->where` I think it should be `Photo::where()` no ?

Comment (hermzz) [reply app/Photo.php]: Neither am I, but I tested it and it worked? Let me do some more digging.

Comment (hermzz) [reply app/Photo.php]: I found this series of [best practices](https://github.com/alexeymezenin/laravel-best-practices#fat-models-skinny-controllers) that has a relevant example. The rest of the tips are reasonable as well.

Comment (ildyria) [reply app/Photo.php]: Hooo nice :+1: 
I basically did bad in every examples. :x

**Pull request merged**